pkg/doozer/prepare/extract.go

```diff
@@ -0,0 +1,95 @@
+package prepare
```

Comment:
Det bør opprettes en sak på at slike funksjoner flyttes ut til en utils pakke

Reply:
Opprettet en sak for refaktorering av duplisert kode (AOT-718)